index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Náš tým</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+
+
+    <h1>Náš tým</h1>
+
+    <div class="zeny">
+
+        <div class="hanka">
+            <div class="image">
+                <img class="foto" src="images/hanka.jpg" alt="Jarmila">

```

Comment:
Obrázky se nezobrazují, protože je uvedená špatná cesta k souboru. Teď se díváme do stejné složky, jako je index.html a hledáme složku images a v ní soubor hanka.jpg. Ten tu ale není. Tady to bylo trošku nešťastně zadané a mělo se pracovat spíš uvnitř podsložky `projekt-03-kontakty-main` a pak by Ti cesta fungovala, takže chyba na naší straně 😔
V tomhle kontextu by správná cesta byla `projekt-03-kontakty-main/projekt-03-kontakty-main/images/hanka.jpg`